src/shogun/neuralnets/DeepAutoencoder.cpp

```diff
@@ -322,5 +322,5 @@ void DeepAutoencoder::init()
 	SG_ADD(&m_do_pretrain, "do_pretrain",
 	    "Whether to pretrain with relevant parameters");
 
-	SG_ADD(&m_sigma, "m_sigma", "Initialization Sigma");
+	SG_ADD(&m_sigma, "sigma", "Initialization Sigma");
```

Comment:
here you change the name, without using a string constant. Also this breaks a unit test (see CI output)

Reply:
https://dev.azure.com/shogunml/shogun/_build/results?buildId=3857&view=logs&j=be3b03d7-4089-5ab3-9462-9af4d8b1dcdf&t=92f9e07a-f827-5326-7f00-9de226b0894b&l=1538

Reply:
Gotcha let me fix that

Reply:
Only removed all m_prefix from SigmoidCalibration, let's see where it fails (or if it does) and I can add small fixes incrementally

Reply:
btw you can also run the unit tests locally using `make test` (details see our developing readme)